src/app/content/constants.ts

```diff
@@ -18,3 +19,11 @@ export const highlightStyles: Array<{label: HighlightColorEnum, passive: string,
   {label: HighlightColorEnum.Purple, passive: '#cbcfff', focused: '#545ec8'},
   {label: HighlightColorEnum.Pink, passive: '#ffc5e1', focused: '#de017e'},
 ];
+
+export const getBaseColors = () => {
```

Comment:
@TomWoodward is there a theme utils file where this should live?

Reply:
it seems to me like this could be implemented as a `getter` somewhere in the `color` object - `theme.color.primary.baseColors`